compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -13510,8 +13510,8 @@ OMR::Z::TreeEvaluator::primitiveArraycopyEvaluator(TR::Node* node, TR::CodeGener
       TR_ASSERT_FATAL(TR::Compiler->target.is64Bit(),
                  "GSCS: Guarded Load OOL Path only defined in 64 bit mode");
       TR::Register *vmReg = cg->getMethodMetaDataRealRegister();
-      int32_t offsetOfConcurrentScavengerActiveByte = offsetof(J9VMThread, privateFlags) + 5;
-      TR::MemoryReference *privFlagMR = generateS390MemoryReference(vmReg, offsetOfConcurrentScavengerActiveByte, cg);
+      TR::MemoryReference *privFlagMR = generateS390MemoryReference(vmReg,
+            TR::Compiler->vm.thisThreadGetConcurrentScavengeActiveByteAddressOffset(cg->comp()), cg);
```

Comment:
I think this might not build with non-OpenJ9 runtimes as the code references J9VMThread. I changed it to use the helper I added in OMEVMEnv.hpp. Please let me know if we want to do it some other way.

Reply:
The whole section was guarded by an `#ifdef` and is waiting to be pulled up to OpenJ9, but this is a welcome addition non the less. Thanks!